packages/account/PasskeyAccount/passkeyOwnerP256.test.ts

```diff
@@ -1,10 +1,12 @@
+import { toUtf8Bytes } from "ethers"
+
 import { PasskeyOwnerP256 } from "./passkeyOwnerP256"
 
 describe("PasskeyOwnerP256", () => {
   const owner = new PasskeyOwnerP256()
 
   it("should sign challenge", async () => {
-    const challenge = "challenge"
+    const challenge = toUtf8Bytes("challenge")
```

Comment:
Q: Why it didn't need this conversion before?

Reply:
The 'challenge' fails ethers.isBytesLike().
However, toUtf8Bytes('challenge') is no longer used after removing the cryptography util.